big-practice/src/scripts/services/customer-service.js

```diff
@@ -1,11 +1,66 @@
-import {BASE_URL} from "../../constants/url";
+import { BASE_URL } from '../../constants/url';
 
 class CustomerService {
   getListCustomer = async () => {
     const apiRequest = await fetch(`${BASE_URL}customers`);
     const dataReponse = await apiRequest.json();
     return dataReponse;
-  };
+  }
+
+  async createCustomer(customer) {
+    const response = await fetch(`${BASE_URL}customers`, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+      },
+      body: JSON.stringify(customer),
+    })
+    if (response.ok) {
+      return response.json();
+    } else {
+      throw new Error('Error creating customer');
+    }
+  }
+
+  validateForm() {
+    var nameInput = document.getElementById('name');
```

Comment:
should use `let` instead of `var`. Using `let` will make debugging easier, avoid hoisting.